.devcontainer/postCreateCommand.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env sh
+set -eux
+
+TARGETOS=linux
+TARGETARCH=amd64
+
+# Install Kind
+curl -L -o kind "https://kind.sigs.k8s.io/dl/v0.22.0/kind-linux-${TARGETARCH}" && chmod +x ./kind
+sudo mv ./kind /usr/local/bin
+
+# Install kubectl
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/${TARGETARCH}/kubectl && chmod +x ./kubectl
+sudo mv ./kubectl /usr/local/bin
+
+# Install helm
+bash -c "curl -s https://get.helm.sh/helm-v3.14.1-linux-${TARGETARCH}.tar.gz > helm3.tar.gz" && tar -zxvf helm3.tar.gz linux-${TARGETARCH}/helm && chmod +x linux-${TARGETARCH}/helm
+sudo mv linux-${TARGETARCH}/helm /usr/local/bin && rm helm3.tar.gz && rm -R linux-${TARGETARCH}
+
+# Install kubebuilder
```

Comment:
I thought, this was a turnkey solution to setup idp on local machine. What exactly are we planning to build using `kubebuilder` ?

Reply:
The repo is to build idpbuilder, idpbuilder uses control-runtime. I don't think we need this I will remove